src/app/service/eco-news/eco-news.service.ts

```diff
@@ -27,4 +29,26 @@ export class EcoNewsService {
       }
     );
   }
+
+  public sortLastThreeNewsChronologically(): any {
+    this.http.get<EcoNewsDto>(`${this.backEnd}econews`)
+      .pipe(take(1))
+      .subscribe(
+        (newsList: EcoNewsDto) => {
+          this.onSortLastThreeNewsFinished(newsList.page, this.selectedId);
+        }
+     );
+  }
+
+  private onSortLastThreeNewsFinished(data: EcoNewsModel[], id: number): void {
+    const separetedNews: Array<EcoNewsModel> = [...data]
+      .sort((a: any, b: any) => {
```

Comment:
set appropriate types instead of using 'any' and rename 'a' and 'b' 'cause it's not clear

Reply:
fixed